package.json

```diff
@@ -11,6 +11,8 @@
 		"prepare": "husky install"
 	},
 	"dependencies": {
+		"@headlessui/react": "^1.6.6",
+		"@heroicons/react": "^1.0.6",
```

Comment:
You can remove this dep and import herocions from the already installed `react-icons` package.